app/assets/javascripts/govuk_publishing_components/load-analytics.js

```diff
@@ -179,11 +179,13 @@ window.GOVUK.loadAnalytics = {
   },
 
   startGa4Modules: function () {
-    var analyticsModules = window.GOVUK.analyticsGa4.analyticsModules
-    for (var property in analyticsModules) {
-      var module = analyticsModules[property]
-      if (typeof module.init === 'function') {
-        module.init()
+    if (!window.GOVUK.analyticsGa4.vars.disableModules) {
```

Comment:
Would it be possible to put a comment here to explain what `disableModules` is used for?